raster_loader/io.py

```diff
@@ -413,11 +417,17 @@ def rasterio_windows_to_records(
         bounds_polygon = shapely.Polygon(shapely.wkt.loads(bounds_geog))
         center_coords = list(*bounds_polygon.centroid.coords)
         center_coords.append(resolution)
+        band_metadata = {
+            "name": band,
+            "type": str(band_type),
+            "nodata": raster_dataset.nodata,
```

Comment:
This field is not part of the spec, should be removed

Reply:
ok, but only because geotiff force to have a single nodata for all bands

Reply:
yep. We can consider adding nodata per value if it's requested

Reply:
note that adding this later need a partial refactor in AT in RASTER commands that have to be taken into account estimating effort.